src/server/pfs/server/api_server.go

```diff
@@ -24,18 +26,23 @@ var (
 
 type apiServer struct {
 	protorpclog.Logger
-	driver drive.Driver
+	driver   drive.Driver
+	reporter *metrics.Reporter
 }
 
-func newAPIServer(driver drive.Driver) *apiServer {
+func newAPIServer(driver drive.Driver, reporter *metrics.Reporter) *apiServer {
 	return &apiServer{
-		Logger: protorpclog.NewLogger("pfs.API"),
-		driver: driver,
+		Logger:   protorpclog.NewLogger("pfs.API"),
+		driver:   driver,
+		reporter: reporter,
 	}
 }
 
 func (a *apiServer) CreateRepo(ctx context.Context, request *pfs.CreateRepoRequest) (response *google_protobuf.Empty, retErr error) {
 	func() { a.Log(request, nil, nil, 0) }()
+	finalMetrics := metrics.ReportUserAction(ctx, a.reporter, "CreateRepo")
```

Comment:
The returned value is used in the defer function. I can re-name it `reportFinalMetrics` to make that clearer